### DISCUSSION TITLE
Graphic Assets

### DISCUSSION THREAD

**Discussion by AlexJ2017** [Programmiermethoden/Dungeon#10] [Ideas]
# Grafische Assets
In der folgenden Tabelle werden Assets gelistet, für die eine Grafik/Animation benötigt wird:
| Asset | Größe (in Tiles) | Varianten | Animationen |
|---|---|---|---|
| Druckplatte |  1x1 |  | Auslösen |
| Hebel |  1x1 |  | Umlegen |
| Schalter | 1x1 |  | Betätigen |
| Schlüssel | Item | Eisen, Gold |  |
| Truhe | 1x1 | Holz, Gold | Öffnen |
| Tür | 1x2 | Holz, Eisen-beschlagen | Öffnen |
| Pfreilschussanlage | 1x1 |  |  |
| Pfeil | Item / 1x1 |  |  |
| Rollender Fels | 2x2 |  | Rollen |
| Stachelfalle | 1x1 |  | Stacheln ausfahren |
| Brunnen | 2x2 | Heilung, Mana, leer |  |
| Munitionskiste | Item |  |  |
| Teleporter | 1x1 | verschiedene Farben | Teleport |
| Trank | 1x1 | verschiedene Farben: Rot (Heilung), Blau (Mana) |  |
| Held | 1x1 | Ritter, Schütze, Magier, Mönch, Drache | Idle, Laufen, Angreifen/Zaubern |
| Lebensanzeige | HUD | verschiedene Füllstände |  |
| Manaanzeige | HUD | verschiedene Füllstände |  |
| (Ausdaueranzeige) | HUD | verschiedene Füllstände |  |
| Ehrfahrungsbalken | HUD | verschiedene Füllstände |  |
| Item Quickbar | HUD | Ausgewählt, nicht ausgewählt |  |
| Sprechblase | HoverText | Skalierbar auf Text |  |
| Dialogbox | HUD | Scrollbar |  |
| Charakterbild | HUD | Held, NPC |  |
| Quest Rahmen | HUD | Skalierbar auf Text |  |
| Quest Checkbox | HUD | Mit Haken, ohne Haken |  |
| Boden | 1x1 | Steinplatten (normal, verwittert), Erde (mit/ohne Wurzeln) |  |
| Wand | 1x1 | Steinziegel (normal, verwittert, mit Fackel), Erde (mit/ohne Wurzeln), |  |
| Wand Ecke | 1x1 | Steinziegel, Erde |  |
| Säule | 1x1 | Stein, Wurzelstrang |  |
| Loch im Boden | 1x1 | Rand (Steinplatten, Erde) |  |

**Comment by AMatutat** [comment]
@AHeinisch @Lena241 Falls ihr beim Implementieren merkt, dass ihr noch Assets braucht, sagt Bescheid. Wir planen im nächsten Jahr uns extra Texturen anfertigen zu lassen.
